demo/index.html

```diff
@@ -20,7 +23,62 @@
       <h3>Basic mtz-ajax-interceptor demo</h3>
       <demo-snippet>
         <template>
-          <mtz-ajax-interceptor></mtz-ajax-interceptor>
+          <test-el></test-el>
+          <dom-module id="test-el">
+            <template>
+              <mtz-ajax-interceptor>
+                <mtz-status-code-interceptor
+                  status="200"
+                  on-intercept="statusCodeInterceptor"
+                ></mtz-status-code-interceptor>
+                <template is="dom-if" if="[[headerInterceptor]]" restamp>
+                  <mtz-status-code-interceptor
+                    status="200"
+                    on-intercept="statusCodeInterceptor"
+                  ></mtz-status-code-interceptor>
+                  <mtz-presend-header-interceptor
+                    key="header"
+                    value="test"
+                  ></mtz-presend-header-interceptor>
+                </template>
+
+                <iron-ajax url="/fake" id="ajax" bubbles></iron-ajax>
+              </mtz-ajax-interceptor>
+              <p>
+                [[statusCodeIntercepted]]
+              </p>
+              <label>
+                Inject additional interceptors
+                <input type="checkbox" checked="{{headerInterceptor::change}}">
+              </label>
+              <br>
+              <button on-tap="intercept">Trigger Interceptors</button>
```

Comment:
@haydenmcp it does also operate on mobile devices. I would say for now on hybrid we use on-click for ease. We can look back into it for 2.0.